integration-tests/bean/src/main/java/org/apache/camel/quarkus/component/bean/BeanResource.java

```diff
@@ -142,4 +142,27 @@ public String parameterBindingAnnotations(String statement, @PathParam("greeting
                 String.class);
     }
 
+    @Path("/produceInterface")
+    @POST
+    @Consumes(MediaType.TEXT_PLAIN)
+    @Produces(MediaType.TEXT_PLAIN)
+    public String produceInterface(String payload) {
+        produceInterface.sayHello(payload);
```

Comment:
Couldn't we simply `return produceInterface.sayHello(payload);` as this is direct under the hood ?

Reply:
Hm... you made me thinking. The purpose of the test is to make sure that the message goes via the Camel route starting with `direct:produceInterface`. Would there be any certainty if we just returned `sayHello(payload)`? I guess yes, but to be really sure, we'd have to add some transformation of the message to the route so that we see that it really passed it.

Reply:
I can try it in the next iteration, when implementing `@Consume`

Reply:
I have added a commit in https://github.com/apache/camel-quarkus/pull/2586 implementing your proposal, @aldettinger. Thanks for suggesting it!